server/server.py

```diff
@@ -1,24 +1,18 @@
 # server.py
-from flask import Flask, render_template, request
+from flask import Flask, render_template, request, url_for
 import speak
 
-# text = "AI AI minha machadinha !!"
-# lang = "pt"
-
 app = Flask(__name__, static_folder="../static/dist", template_folder="../static")
 @app.route("/", methods=["GET","POST"])
 def index():
     return render_template("index.html")
 
-@app.route("/hello", methods=["GET","POST"])
-def hello():
-    text=''
-    lang=''
-    if request.method == "POST":
-        text = request.form["text"]
-        lang = request.form["lang"]
-        print(text)
-    return speak.get_speak(text,lang)
+@app.route("/hello/<text>/<lang>")
+def hello(text,lang):
+    if request.method == "GET":
+        speak.get_speak(text,lang)
+        return url_for("static", filename="mp3_path")
```

Comment:
BTW `"mp3_path"` is meant to be a variable that we will get from  `get_speak` **Example**: `mp3_path = speak.get_speak()`